gax/BUILD.bazel

```diff
@@ -16,29 +16,44 @@ package(default_visibility = ["//visibility:public"])
 
 licenses(["notice"])  # Apache 2.0
 
+# Note: googleapis doesn't define a cc_proto_library for operations_proto,
+# so declare one here ourselves.
+cc_proto_library(
+    name = "longrunning_cpp_proto",
+    deps = ["@com_google_googleapis//google/longrunning:operations_proto"],
+)
+
 cc_library(
     name = "gax",
     srcs = [
         "backoff_policy.cc",
         "call_context.cc",
         "internal/gtest_prod.h",
+        "operations_client.cc",
+        "operations_stub.cc",
         "status.cc",
     ],
     hdrs = [
         "backoff_policy.h",
         "call_context.h",
         "retry_policy.h",
+        "operation.h",
+        "operations_client.h",
+        "operations_stub.h",
         "status.h",
         "status_or.h",
     ],
     deps = [
         "@com_github_grpc_grpc//:grpc++",
+        ":longrunning_cpp_proto",
     ],
 )
 
 gax_unit_tests = [
     "backoff_policy_unittest.cc",
     "call_context_unittest.cc",
+    "operation_unittest.cc",
```

Comment:
For what is worth, other libraries use `*_test.cc`, I think that is also the practice inside Google, and `clang-format(1)` understands that the first include in `foo_test.cc` should be `foo.h`.

Reply:
I'll file an issue to fix the names.

Reply:
Tracked by #39